pkg/controller/master/setting/support_bundle_image.go

```diff
@@ -4,94 +4,46 @@ import (
 	"encoding/json"
 	"reflect"
 
-	"github.com/mitchellh/mapstructure"
-	catalogv1api "github.com/rancher/rancher/pkg/apis/catalog.cattle.io/v1"
 	"github.com/sirupsen/logrus"
-	"helm.sh/helm/v3/pkg/chart"
-	"helm.sh/helm/v3/pkg/chartutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 
 	"github.com/harvester/harvester/pkg/generated/controllers/harvesterhci.io/v1beta1"
 	"github.com/harvester/harvester/pkg/settings"
+	"github.com/harvester/harvester/pkg/util"
+	utilHelm "github.com/harvester/harvester/pkg/util/helm"
 )
 
 const (
 	SupportBundleRepository = "support-bundle-kit"
```

Comment:
Do we still need this const? as the old code has been removed.

Reply:
Removed it. Thanks.